src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.WebAssembly.cs

```diff
@@ -119,6 +125,34 @@ private unsafe int JsIndexOfCore(ReadOnlySpan<char> source, ReadOnlySpan<char> t
             return idx;
         }
 
+        private char[] emptyCharsToRemove = {
```

Comment:
"Unicode category"? What category are empty unicode chars in?

Reply:
why specifically these characters ?

Reply:
Because they are "empty". The name is not descriptive enough? Another way would be "formattingChars" or "controlChars" but empty sounds more reasonable when it comes to removal

Reply:
Or we can do:
// remove chars that are ignored by ICU hashing algorithm but not ignored by invariant hashing

Reply:
`// chars that are ignored by locale equals algorithm but not ignored by invariant hashing`

Is that true ?

Reply:
Yes, it is but after re-evaluation: there's 943 chars of this type that should be ignored.